src/main/java/mate/academy/service/UserService.java

```diff
@@ -3,18 +3,25 @@
 import mate.academy.model.User;
 
 public class UserService {
-    private static final User[] users = new User[] {
+    private static final User[] users = new User[]{
             new User("bob@i.ua", "1234"),
             new User("alice@i.ua", "1234")
     };
 
     /**
      * Find user by email. All users are stored in <code>private static final User[] users</code>
+     *
      * @param email - the input parameter
      * @return - user if his email is equal to passed email.
      * Return <code>null</code> if there is no suitable user
      */
     public User findByEmail(String email) {
-        return null;
+        User returnUser = new User("USER NOT CONTAIN", "USER NOT CONTAIN");
+        for (User user : users) {
+            if (email.equals(user.getEmail())) {
+                returnUser = user;
+            }
+        }
+        return returnUser;
```

Comment:
Return <code>null</code> if there is no suitable user

Reply:
but when I add NULL, tests fail